src/enzax/sbml.py

```diff
@@ -65,11 +65,35 @@ def sbml_to_sympy(model):
         )
         for r in reactions_sbml
     ]
-    return reactions_sympy
+
+    output = {"reactions": reactions_sympy}  # Store output in a dictionary
+
+    assignments_sbml = model.getListOfRules()
+    if assignments_sbml:
+        assignments_sympy = {
+            a.variable: SBMLMathMLParser().parse_str(
+                libsbml.writeMathMLToString(
+                    libsbml.parseL3Formula(
+                        libsbml.formulaToL3String(a.getMath())
+                    )
+                )
+            )
+            for a in assignments_sbml
+        }
+
+        output["assignments"] = assignments_sympy
+
+    return output
 
 
 def sympy_to_enzax(reactions_sympy):
```

Comment:
It might be cleaner if this function always returned a tuple of (symbolic module, assignments), with the assignments just being empty in the case that there aren't any of them. Alternatively I guess there could be a separate function for getting the assignments?